tests/test_pipelines_token_classification.py

```diff
@@ -190,6 +191,329 @@ def _test_pipeline(self, nlp: Pipeline):
             ],
         ]
 
+        ungrouped_inputs_all_scores = [
```

Comment:
Should this be in a fixture too?

Reply:
Good point! Moved to json fixture along w/ the already existing inputs above this.